application/models/claim.go

```diff
@@ -895,9 +895,30 @@ func (c *Claim) UpdateStatus(ctx context.Context, newStatus api.ClaimStatus) err
 	return nil
 }
 
+// Based on the claim's UpdatedAt field, unless there is a
+//   ClaimHistory for this claim that is for a Status Update where tne
+//   new field is Review1.  Uses the CreatedAt time of the earliest history
+//   with Status change to Review1.
 func (c *Claim) SubmittedAt(tx *pop.Connection) time.Time {
-	// TODO: use the history table to get the real date
-	return c.UpdatedAt
+	var histories ClaimHistories
+
+	err := tx.RawQuery(`
+SELECT created_at
```

Comment:
If you're concerned about extraneous whitespace, you can do this:
```suggestion
	err := tx.RawQuery("SELECT created_at"+
		" FROM claim_histories"+
		" WHERE claim_id = ? AND field_name = ? AND action = ? AND new_value = ?"+
		" ORDER BY created_at ASC",
```
otherwise, just indent each line a couple places to line up with the surrounding code.

Reply:
:+1: